etc/non-ecs-schema.json

```diff
@@ -10,7 +10,8 @@
   "winlogbeat-*": {
     "winlog.event_data.OriginalFileName": "keyword",
     "winlog.event_data.GrantedAccess": "keyword",
-    "winlog.event_data.CallTrace": "keyword"
+    "winlog.event_data.CallTrace": "keyword",
+	 "winlog.event_data.TargetImage": "keyword"
```

Comment:
I don't see this in the integration mapping - can you point to a source or screenshot to verify it is a keyword

Reply:
Could not find it on the index templates also...

Reply:
@Samirbous what's the output of `GET /logs-*/_mapping/field/winlog.event_data.TargetImage`?

Reply:
![image](https://user-images.githubusercontent.com/64742097/137491913-a0ff26e1-2bc0-42f4-96e7-4fa1761a32aa.png)